adafruit_as7341.py

```diff
@@ -640,30 +656,30 @@ def _set_smux(self, smux_addr, smux_out1, smux_out2):
         self._write_register(smux_addr, smux_byte)
 
     @property
-    def gain(self):
+    def gain(self) -> Gain:
```

Comment:
Similar to the way it worked above this should return the same type as the associated `CV`'s `value` parament in that tuple - so here it should return `int`

Reply:
This will be true for all the getter methods like this that are for `CV`'s!